src/solvers/dgmulti/flux_differencing_gauss_sbp.jl

```diff
@@ -308,13 +308,36 @@ end
   end
 end
 
-# Specialized constructor for GaussSBP approximation type on quad elements. Restricting to
-# VolumeIntegralFluxDifferencing for now since there isn't a way to exploit this structure
-# for VolumeIntegralWeakForm yet.
+function DGMulti(element_type::Line,
+                 approximation_type::GaussSBP,
+                 volume_integral, surface_integral;
+                 polydeg::Integer,
+                 surface_flux=flux_central,
+                 kwargs...)
+
+  # explicitly specify Gauss quadrature rule with polydeg+1 points
+  rd = RefElemData(element_type, Polynomial(), polydeg,
```

Comment:
Okay so just to be sure I am interpreting the comments here correctly. We pass `Polynomial()` discretization type, which is the modal one with an available constructor, to make some of the matrices and then some data is overwritten after the construction to create a nodal Gauss SBP scheme. Is that the correct line of thinking?

Reply:
Yup! That's precisely it.